src/components/pages/Data.tsx

```diff
@@ -28,6 +25,13 @@ export default function Data() {
         })
     }
 
+    const getAllRecords = async () => {
+        const api = new httpClient()
+        // TODO: Figure out a better place to put this so we don't keep updating this either 
```

Comment:
could have a "data" section in the context as well, we may eventually put filters in the data tab. that can happen after release tho

Reply:
I'm just gonna leave this here for now because of how many records we would have to store in context that would otherwise bog down the app